readme.md

```diff
@@ -1322,6 +1322,7 @@
 | [devices.css](http://marvelapp.github.io/devices.css/) | Pure CSS phones and tablets by Marvel App. |
 | [Troopl](https://troopl.com) | Build and publish a free portfolio in minutes. |
 | [Apifox](https://apifox.cn) | Apifox = Postman + Swagger + Mock + JMeter |
+| [okso.app](https://okso.app) | The drawing app with a nested "drawing-inside-the-drawing" structure |
```

Comment:
Awesome, Thanks for the PR!!

Can you please put it under #online-design-tools section?

Reply:
Sure, moved the link and updated the PR 👌🏻